tests/system/action/user/test_save_saml_account.py

```diff
@@ -448,3 +448,185 @@ def test_create_saml_account_boolean_integer_types_error(self) -> None:
+                    "saml_enabled": True,
+                    "saml_attr_mapping": {
+                        "saml_id": "username",
+                        "title": "title",
+                        "first_name": "firstName",
+                        "last_name": "lastName",
+                        "email": "email",
+                        "gender": "gender",
+                        "pronoun": "pronoun",
+                        "is_active": "is_active",
+                        "is_physical_person": "is_person",
+                        "meeting": {
+                            "external_id": "Landtag",
+                            "external_group_id": "Delegates",
+                        },
+                    },
+                },
+            }
+        )
+        self.create_meeting()
+        self.set_models(
+            {
+                "group/1": {"external_id": "Default"},
+                "group/2": {"external_id": "Delegates"},
+                "group/3": {"external_id": "Admin"},
+                "meeting/1": {"external_id": "Landtag", "default_group_id": 1},
+                "user/1": {"saml_id": "admin_saml"},
+            }
+        )
+
+    def test_create_user_with_membership(self) -> None:
+        response = self.request("user.save_saml_account", {"username": ["111"]})
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/2",
+            {
+                "saml_id": "111",
+                "username": "111",
+                "meeting_user_ids": [1],
+                "meeting_ids": [1],
+            },
+        )
+        self.assert_model_exists(
+            "meeting_user/1", {"user_id": 2, "group_ids": [2], "meeting_id": 1}
+        )
+        self.assert_model_exists(
+            "group/2", {"meeting_user_ids": [1], "external_id": "Delegates"}
+        )
+
+    def test_update_user_with_membership(self) -> None:
+        response = self.request("user.save_saml_account", {"username": ["admin_saml"]})
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/1",
+            {
+                "saml_id": "admin_saml",
+                "username": "admin",
+                "meeting_user_ids": [1],
+                "meeting_ids": [1],
+            },
+        )
+        self.assert_model_exists(
+            "meeting_user/1", {"user_id": 1, "group_ids": [2], "meeting_id": 1}
+        )
+        self.assert_model_exists(
+            "group/2", {"meeting_user_ids": [1], "external_id": "Delegates"}
+        )
+
+    def test_create_user_invalid_meeting(self) -> None:
+        """silent fail, user created and logged in"""
+        organization = self.get_model("organization/1")
+        for key in ("meta_deleted", "meta_position"):
+            organization.pop(key, None)
+        organization["saml_attr_mapping"]["meeting"]["external_id"] = "Kreistag"
+        self.set_models({"organization/1": organization})
```

Comment:
Cleaner solution: save the `saml_attr_mapping` into its own variable in the `setUp` and modify it here and simply update the respective field in the organization

(same for the other tests)

Reply:
done